pkg/api/customization/monitor/cluster_graph_action.go

```diff
@@ -248,9 +248,12 @@ func convertInstance(seriesSlice []*TimeSeries, nodeMap map[string]string, resou
 		name := v.Name
 
 		if resourceType == ResourceCluster || resourceType == ResourceNode || resourceType == ResourceAPIServer || resourceType == ResourceEtcd || resourceType == ResourceScheduler || resourceType == ResourceControllerManager || resourceType == ResourceFluentd || resourceType == ResourceIngressController {
-			hostName := strings.Split(v.Tags["instance"], ":")[0]
-			if v.Name != "" && nodeMap[hostName] != "" {
-				name = strings.Replace(v.Name, v.Tags["instance"], nodeMap[hostName], -1)
+			hostAndPort := strings.Split(v.Tags["instance"], ":")
```

Comment:
what is this fix for? Doesn't seem to do anything with the owner reference.

Reply:
this pr also add more check for array in case similar panic happen

Reply:
I don't think it is necessary. `strings.Split` will always return a string in the slice. Even an empty string will also be returned.